lib/cog/adapters/hipchat.ex

```diff
@@ -20,7 +20,8 @@ defmodule Cog.Adapters.HipChat do
   end
 
   def lookup_user(opts) do
-    HipChat.API.lookup_user(opts)
+    {:ok, user} = HipChat.API.lookup_user(opts)
+    {:ok, Map.put(user, :id, to_string(user.id))}
```

Comment:
The `ChatProvider.chat_provider_user_id` expects a string, while our JSON parsing of this id gives us an integer. It's not really a concern of this adapter so it doesn't fit here, but this is the easiest place to add it for now, since it's the only adapter that doesn't give us a string id.